app/views/tracks.js

```diff
@@ -11,6 +11,109 @@ const TracksView = Mn.CompositeView.extend({
 
     childView: TrackView,
 
+    events: {
+         'mousedown .drag': 'dragStart'
+    },
+
+    initialize() {
+        this.dragObj = {
+            'dragDown': false,
+            'model': undefined,
+            'newIndex': undefined
+        };
+        this.listenTo(this.collection, 'sync', this.bindDragEvents);
+    },
+
+    bindDragEvents() {
+        $(document).mousemove(this.drag.bind(this));
+        $(document).mouseup(this.dragEnd.bind(this));
+    },
+
+    dragStart(e) {
+        e.stopPropagation();
+        let htmlTrack = $(e.currentTarget).parents('li');
+        htmlTrack.addClass('dragged');
+        let id = htmlTrack.attr('data-id');
```

Comment:
It's more readable to use https://api.jquery.com/data/#data2 to access a data-attribute.